src/Libraries/RevitNodes/Elements/SunSettings.cs

```diff
@@ -143,8 +170,165 @@ internal static DateTime TranslateTime(DateTime utc)
+        { "Asia/Seoul", "Korea Standard Time" },
+        { "Asia/Shanghai", "China Standard Time" },
+        { "Asia/Singapore", "Singapore Standard Time" },
+        { "Asia/Taipei", "Taipei Standard Time" },
+        { "Asia/Tashkent", "West Asia Standard Time" },
+        { "Asia/Tbilisi", "Georgian Standard Time" },
+        { "Asia/Tehran", "Iran Standard Time" },
+        { "Asia/Tokyo", "Tokyo Standard Time" },
+        { "Asia/Ulaanbaatar", "Ulaanbaatar Standard Time" },
+        { "Asia/Vladivostok", "Vladivostok Standard Time" },
+        { "Asia/Yakutsk", "Yakutsk Standard Time" },
+        { "Asia/Yekaterinburg", "Ekaterinburg Standard Time" },
+        { "Asia/Yerevan", "Armenian Standard Time" },
+        { "Atlantic/Azores", "Azores Standard Time" },
+        { "Atlantic/Cape_Verde", "Cape Verde Standard Time" },
+        { "Atlantic/Reykjavik", "Greenwich Standard Time" },
+        { "Australia/Adelaide", "Cen. Australia Standard Time" },
+        { "Australia/Brisbane", "E. Australia Standard Time" },
+        { "Australia/Darwin", "AUS Central Standard Time" },
+        { "Australia/Hobart", "Tasmania Standard Time" },
+        { "Australia/Perth", "W. Australia Standard Time" },
+        { "Australia/Sydney", "AUS Eastern Standard Time" },
+        { "Etc/GMT", "UTC" },
+        { "Etc/GMT+11", "UTC-11" },
+        { "Etc/GMT+12", "Dateline Standard Time" },
+        { "Etc/GMT+2", "UTC-02" },
+        { "Etc/GMT-12", "UTC+12" },
+        { "Europe/Amsterdam", "W. Europe Standard Time" },
+        { "Europe/Athens", "GTB Standard Time" },
+        { "Europe/Belgrade", "Central Europe Standard Time" },
+        { "Europe/Berlin", "W. Europe Standard Time" },
+        { "Europe/Brussels", "Romance Standard Time" },
+        { "Europe/Budapest", "Central Europe Standard Time" },
+        { "Europe/Dublin", "GMT Standard Time" },
+        { "Europe/Helsinki", "FLE Standard Time" },
+        { "Europe/Istanbul", "GTB Standard Time" },
+        { "Europe/Kiev", "FLE Standard Time" },
+        { "Europe/London", "GMT Standard Time" },
+        { "Europe/Minsk", "E. Europe Standard Time" },
+        { "Europe/Moscow", "Russian Standard Time" },
+        { "Europe/Paris", "Romance Standard Time" },
+        { "Europe/Sarajevo", "Central European Standard Time" },
+        { "Europe/Warsaw", "Central European Standard Time" },
+        { "Indian/Mauritius", "Mauritius Standard Time" },
+        { "Pacific/Apia", "Samoa Standard Time" },
+        { "Pacific/Auckland", "New Zealand Standard Time" },
+        { "Pacific/Fiji", "Fiji Standard Time" },
+        { "Pacific/Guadalcanal", "Central Pacific Standard Time" },
+        { "Pacific/Guam", "West Pacific Standard Time" },
+        { "Pacific/Honolulu", "Hawaiian Standard Time" },
+        { "Pacific/Pago_Pago", "UTC-11" },
+        { "Pacific/Port_Moresby", "West Pacific Standard Time" },
+        { "Pacific/Tongatapu", "Tonga Standard Time" }
```

Comment:
seems like maybe we are missing some mappings...http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/zone_tzid.html